tap_outbrain/__init__.py

```diff
@@ -275,9 +275,8 @@ def sync_campaign_page(state, access_token, account_id, campaign_page, selected_
     campaigns = [parse_campaign(campaign) for campaign
                  in campaign_page.get('campaigns', [])]
 
-    campaign_sub_streams = streams.SUB_STREAMS.get('campaign')
     if (
-        streams.CampaignPerformance.name not in campaign_sub_streams
+        not (hasattr(streams.CampaignPerformance, 'parent') and streams.CampaignPerformance.parent == 'campaigns')
         or streams.CampaignPerformance.name not in selected_streams
```

Comment:
Do we really need this change? IMO below is sufficient.

```suggestion
        streams.CampaignPerformance.name not in selected_streams
```

Reply:
Yes, your changes look good. 👍
I had made the updates for keeping the original code flow intact.
Have made the changes to simplify things with just the minimum checks needed.